### ISSUE TITLE
Missing sdist for 0.8.21

### ISSUE THREAD

**Issue by mgorny** [holgern/py-scrypt#16]
I see you're probably working on it but filing this for tracking purposes.

**Comment by brianoflondon** [comment]
Adding this here If this is what is causing the following on an older intel mac:

```
  • Updating scrypt (0.8.20 -> 0.8.21): Failed

  RuntimeError

  Unable to find installation candidates for scrypt (0.8.21)

  at ~/Library/Application Support/pypoetry/venv/lib/python3.11/site-packages/poetry/installation/chooser.py:73 in choose_for
       69│ 
       70│             links.append(link)
       71│ 
       72│         if not links:
    →  73│             raise RuntimeError(f"Unable to find installation candidates for {package}")
       74│ 
       75│         # Get the best link
       76│         chosen = max(links, key=lambda link: self._sort_key(package, link))
       77│ 

Cannot install scrypt.
```

**Comment by holgern** [comment]
Fixed in 0.8.22
